### ISSUE TITLE
Rebuild against new Scalafix version

### ISSUE THREAD

**Issue by michaelmior** [pixiv/scalafix-pixiv-rule#46]
I received the following warning when using these rules

```
[info] Loading external rule(s) built against an old version of Scalafix (0.9.33).
[info] This might not be a problem, but in case you run into unexpected behavior, you
[info] should try a more recent version of the rules(s) if available. If that does
[info] not help, request the rule(s) maintainer to build against Scalafix 0.10.4
[info] or later, and downgrade Scalafix to 0.9.x (x>=33) for the time being.
```

**Comment by Javakky-pxv** [comment]
@michaelmior 
Thank you!
We have released version 3.0.1 and hope you will try it.
If you have any further comments, please do not hesitate to raise an issue!
https://github.com/pixiv/scalafix-pixiv-rule/releases/tag/v3.0.1

**Comment by michaelmior** [comment]
@Javakky-pxv Thanks! That seems to have removed the warning.
